locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -338,18 +334,18 @@ This philosophy can lead to some potentially problematic situations.
 Take this snippet for example:
 
 ```js
+let bar;
```

Comment:
Why are you initializing the variable up here instead of assigning it directly as before?